PathOperation.roboFontExt/resources/makePathOperatorIcon.py

```diff
@@ -0,0 +1,19 @@
+size(500, 500)
```

Comment:
add `from drawBot import *` so this script works anywhere in RF when drawBot is installed

Reply:
✔︎ done